src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+
+    background-image: url(../images/star.svg);
+    object-fit: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &--4 > &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
And what if we'll set 3 here? Your stars should have functionality from previous task (if we'll change number in this modifier, it should make needed amount of stars active)